packages/types/v1/profile.ts

```diff
@@ -0,0 +1,30 @@
+import z from "zod";
+import { ZUserNotificationSettings } from "./users";
+import { Role, IdentityProvider, Objective } from "@prisma/client";
+
+export const ZProfileUpdateOutput = z.object({
+  id: z.string(),
+  createdAt: z.date(),
+  updatedAt: z.date(),
+  name: z.string().nullish(),
+  email: z.string(),
+  emailVerified: z.date().nullish(),
+  password: z.string().nullish(),
+  onboardingCompleted: z.boolean(),
+  identityProvider: z.nativeEnum(IdentityProvider),
+  identityProviderAccountId: z.string().nullish(),
+  groupId: z.string().nullish(),
+  role: z.nativeEnum(Role).nullish(),
+  objective: z.nativeEnum(Objective).nullish(),
+  notificationSettings: ZUserNotificationSettings,
+});
+
+export type TProfileUpdateOutput = z.infer<typeof ZProfileUpdateOutput>;
+
+export const ZProfile = z.object({
```

Comment:
I would like to add `createdAt` and `updatedAt` here, as we also need this in the other data models and that makes it more consistent 😊